sdk/communication/azure-communication-chat/src/test/resources/session-records/ChatAsyncClientTest.canCreateNewThreadWithoutSettingRepeatabilityID[1].json

```diff
@@ -3,8 +3,7 @@
     "Method" : "POST",
     "Uri" : "https://REDACTED.communication.azure.com/identities?api-version=2021-03-07",
     "Headers" : {
-      "User-Agent" : "azsdk-java-azure-communication-identity/1.2.0-beta.1 (11.0.7; Windows 10; 10.0)",
-      "x-ms-client-request-id" : "696b1a81-44f9-4da1-a831-c2b6faf36cf5",
```

Comment:
We should still send this. What has changed to remove this header?

Reply:
Nothing has changed, I will try to regenerate the file and see if this persists

Reply:
Seems like an issue with the diff viewer, if you look at the file we are still sending it

Reply:
Sorry, I still only see it in response header but not request header

Reply:
You were right I saw it in the response. Regenerating the files seems to fix this, I don't know what happened on this version